misc.go

```diff
@@ -482,6 +482,7 @@ var tokenMap = map[string]int{
 	"OPTIMIZE":                 optimize,
 	"OPTION":                   option,
 	"OPTIONALLY":               optionally,
+	"OPT_TRACE":                opttrace,
```

Comment:
Seems that we don't need it? Check https://github.com/pingcap/parser/pull/654/files